stub-content/index.js

```diff
@@ -54,6 +61,27 @@ function renderUpdates(ev) {
   if (ev.detail.length) document.querySelector('.update-submit').style.display = 'block';
 }
 
+function renderStatus(ev) {
+  var msgEls = ev.detail.current.map(function(update, i) {
+    var li = document.createElement('li');
+    var check = document.createElement('input');
+    var label = document.createElement('label');
+    check.type = 'checkbox';
+    label.textContent = update;
+
+    li.id = 'installed-' + i;
+    li.appendChild(check);
+    li.appendChild(label);
+
+    return li;
```

Comment:
I should probably test this, but giving it a quick eyeball, should the checkbox element be nested within the `<label>`? or should the label be otherwise linked to the checkbox? Translation: Does clicking on the label element toggle the checkbox?